sale_contact_type/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+Create/modfy a contact and set the address type to 'Ordering address'
+Create a SO and select it in 'Customer' field.
```

Comment:
Don't compact `SO`. This is user documentation. And specify full path to go for creating that sales order.